tests/common/system_utils/docker.py

```diff
@@ -67,8 +67,8 @@ def load_docker_registry_info(dut):
             raise
 
     host = registry_vars.get("docker_registry_host")
-    username = registry_vars.get("docker_registry_username")
-    password = registry_vars.get("docker_registry_password")
+    username = None
+    password = None
```

Comment:
does this work for the internal docker registry as well?

Reply:
revert back.

---
In reply to: [467237544](https://github.com/Azure/sonic-mgmt/pull/2069#discussion_r467237544) [](ancestors = 467237544)